src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,816 @@
+ *   }
+ *
+ *   // Organization created.
+ * });
+ *
+ * @param   {Object}    data     Organization data object.
+ * @param   {Function}  [cb]     Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'create', 'organizations.create');
+
+/**
+ * Get all roles.
+ *
+ * @method    getAll
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example <caption>
+ *   This method takes an optional object as first argument that may be used to
+ *   specify pagination settings. If pagination options are not present,
+ *   the first page of a limited number of results will be returned.
+ * </caption>
+ *
+ * // Pagination settings.
+ * var params = {
+ *   per_page: 10,
+ *   page: 0
+ * };
+ *
+ * management.organizations.getAll(params, function (err, organizations) {
+ *   console.log(organizations.length);
+ * });
+ *
+ * @param   {Object}    [params]          Organizations parameters.
+ * @param   {Number}    [params.per_page] Number of results per page.
+ * @param   {Number}    [params.page]     Page number, zero indexed.
+ * @param   {Function}  [cb]              Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'getAll', 'organizations.getAll');
+
+/**
+ * Get an Auth0 organization.
+ *
+ * @method    get
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * management.organizations.get({ id: ORGANIZATION_ID }, function (err, role) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization);
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'get', 'organizations.get');
+
+/**
+ * Update an existing organization.
+ *
+ * @method    update
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var data = { name: 'New name' };
+ * var params = { id: ORGANIZATION_ID };
+ *
+ * // Using auth0 instance.
+ * management.updateOrganization(params, data, function (err, organization) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization.name);  // 'New name'
+ * });
+ *
+ * // Using the organizations manager directly.
+ * management.organizations.update(params, data, function (err, organization) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization.name);  // 'New name'
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {Object}    data          Updated organization data.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'update', 'organizations.patch');
```

Comment:
We are missing get by name endpoint in the SDK:

`GET https://{{auth0_domain}}/api/v2/organizations/name/:org_name`